modules/rostests/winetests/riched20/editor.c

```diff
@@ -8987,48 +9022,15 @@ static void test_EM_SELECTIONTYPE(void)
     DestroyWindow(hwnd);
 }
 
-static void test_window_classes(void)
```

Comment:
I'm not sure I understand the removal of this test, whereas it exists already in Wine:
https://github.com/wine-mirror/wine/blame/master/dlls/riched20/tests/editor.c#L9202

Reply:
This test was not part of Wine-6.10, but it was present in Wine Staging from that time frame. I have added this test back into this Wine Sync because eventually it was added into Wine. Thanks.